ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/computation/ValidatedCommandFactory.java

```diff
@@ -144,9 +152,19 @@ private static Command createForAlterSystemQuery(
     if (resolvedItem.isPresent()
         && Objects.equals(resolvedItem.get().getPropertyName(), PROCESSING_GUARANTEE_CONFIG)
         && !context.getPersistentQueries().isEmpty()) {
+      final Collection<QueryId> runningQueries =
+          context.getPersistentQueries()
+              .stream()
+              .map(QueryMetadata::getQueryId)
+              .collect(Collectors.toList());
+      LOG.error("Failed to set {} to {} due to the {} persistent queries currently running: {}",
+                propertyName, propertyValue, runningQueries.size(), runningQueries);
       throw new ConfigException(
-          String.format("Failed to set %s to %s. Please terminate all running queries "
-              + "before attempting to set %s", propertyName, propertyValue, propertyName), null);
+          String.format("Unable to set %s to %s due to existing persistent queries. The %s may"
```

Comment:
I think this can be consolidated a bit to something along the lines of:

> ${propertyName} may not be changed for running persistent queries which have already processed data using a different ${propertyName}. To modify ${propertyName} you must first terminate all running persistent queries.

Or a slightly different version:

> ${propertyName} may not be changed for running persistent queries which are already processing data using a different ${propertyName}. To modify ${propertyName} you must first terminate all running persistent queries.